include/truffle.h

```diff
@@ -126,6 +127,8 @@ void truffle_write_idx_b(void *object, int idx, bool value);
 
 // Strings
 void *truffle_read_string(const char *string);
+void *truffle_read_bytes(const char *bytes);
```

Comment:
This is in the string section of the API, but from the name alone it isn't clear it'll read until the first zero byte. Maybe `truffle_read_string_bytes`